<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoCardTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_init">
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_capture">
     <property name="text">
      <string>抓拍</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="checkBox_autoCapture">
     <property name="text">
      <string>自动抓拍</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_video">
     <property name="text">
      <string>显示视频图像</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="comboBox_timer">
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4000</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_capture">
     <property name="text">
      <string>显示抓拍图像</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>间隔时间：（毫秒）</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
